### PR TITLE
Sex up the readme.

### PR DESCRIPTION
**Pull request by jonnyarnold**
Added a little bit more clarification on what it actually is. Fixed some typos and glamourised some sentences.

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **designed for change**, **easy to understand** 
and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [x] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
  tests).
- [x] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Reevoocop
 
-20% MonkeyPatch
-80% YAML
-100% Style
+*20% MonkeyPatch, 80% YAML, 100% Style*
+
+Reevoocop uses [Rubocop](https://github.com/bbatsov/rubocop) to enforce Reevoo's style guidelines.
```
Comment (ionut998) [inline README.md]: Maybe we should write the name as they write it RuboCop in the link

**Comment by ionut998** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (lpil) [inline README.md]: :+1:

**Pull request merged**